### PR TITLE
feat(mongodb): add `payload_template` field for bridge (e5.0)

### PR DESCRIPTION
**Pull request by thalesmg**
https://emqx.atlassian.net/browse/EMQX-8705

Adds a `payload_template` fields that allows users to customize the payload to publish to MongoDB.

### CODE REVIEW

File: lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_mongodb.conf [hunk 1]
```diff
@@ -86,4 +86,15 @@ emqx_ee_bridge_mongodb {
       zh: "桥接名称"
     }
   }
+
+  payload_template {
+    desc {
+      en: "The template for formatting the outgoing messages.  If undefined, will send all the available context in JSON format."
+      zh: "用于格式化外发信息的模板。 如果未定义，将以JSON格式发送所有可用的上下文。"
```
Comment (zmstone) [inline lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_mongodb.conf:93]:
```suggestion
      zh: "用于格式化写入 MongoDB 的消息模板。 如果未定义，规则引擎会使用 JSON 格式序列化所有的可见输入，例如 clientid, topic, payload 等。"
```

File: lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_mongodb.conf [hunk 2]
```diff
@@ -86,4 +86,15 @@ emqx_ee_bridge_mongodb {
       zh: "桥接名称"
     }
   }
+
+  payload_template {
+    desc {
+      en: "The template for formatting the outgoing messages.  If undefined, will send all the available context in JSON format."
```
Comment (zmstone) [inline lib-ee/emqx_ee_bridge/i18n/emqx_ee_bridge_mongodb.conf:92]:
```suggestion
      en: "The template for formatting the outgoing messages.  If undefined, rule engine will use JSON format to serialize all visible inputs, such as clientid, topic, payload etc."
```

**Comment by terry-xiaoyu** [comment]
Looks good, but missing the change logs?

**Review by terry-xiaoyu — APPROVED** [review]

**Comment by thalesmg** [comment]
> Looks good, but missing the change logs?

Just to register what we talked later: since we have not yet release the EE version, we have not been collecting changes for that edition.  :smiley_cat:

**Pull request merged**